ts/deploy.ts

```diff
@@ -138,7 +138,7 @@ export async function deployAll(
     );
     await waitAndRegister(rollup, "rollup", verbose);
 
-    await vault.setRollupAddress(rollup.address);
+    await waitUntilMined(vault.setRollupAddress(rollup.address));
     await waitUntilMined(depositManager.setRollupAddress(rollup.address));
```

Comment:
Would be great if we could batch calls like this but from what I've seen of typechain/ethers it's currently not possible.